### PR TITLE
Adapt dekorate

### PR DESCRIPTION
**Pull request by jacobdotcosta**

### CODE REVIEW

File: run_tests_with_dekorate.sh [hunk 1]
```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/env bash
 
+SB_VERSION_SWITCH=""
```
Comment (Sgitario) [inline run_tests_with_dekorate.sh]: I don't like to update this script only for this example. Instead, I would change all the scripts for all the examples with a more general purpose change: this is allowing to provide any `-Dkey=value` arguments.

File: pom.xml [hunk 2]
```diff
@@ -16,22 +16,22 @@
...
-    <artifactId>snowdrop-dependencies</artifactId>
-    <version>2.5.12.Final</version>
-  </parent>
-
   <groupId>dev.snowdrop.example</groupId>
   <artifactId>rest-http</artifactId>
-  <version>2.5.12-3-SNAPSHOT</version>
-  <name>Spring Boot - HTTP Example</name>
-  <description>Spring Boot - HTTP Example</description>
+  <version>2.11.0-SNAPSHOT</version>
```
Comment (Sgitario) [inline pom.xml]: Same comment about versioning as in other reviews :/

File: pom.xml [hunk 3]
```diff
@@ -16,22 +16,22 @@
...
+  <name>Dekorate - Spring Boot - HTTP Example</name>
+  <description>Dekorate - Spring Boot - HTTP Example</description>
```
Comment (Sgitario) [inline pom.xml]: I don't like to rename the examples adding `Dekorate`. Dekorate is just an utility to generate resources. For example, we also use `helm` in the snowdrop examples.

**Review by Sgitario — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (jacobdotcosta) [reply pom.xml]: fixed.

File: run_tests_with_dekorate.sh (hunk 1 above)
Comment (jacobdotcosta) [reply run_tests_with_dekorate.sh]: Updated the script so it passes all the parameters. Also updated the docs with info on how to override the parameters.

**Review by Sgitario — APPROVED** [review]
lgtm!

**Review by Sgitario — CHANGES_REQUESTED** [review]
Sorry, one additional change. The version of the `spring-maven-plugin` plugin must be provided as we are not inheriting from the snowdrop spring bom parent anylonger.

**Review by Sgitario — APPROVED** [review]
lgtm now, conflicts need to be resolved before merging

**Pull request merged**